Sources/AmplitudeCore/Remote Config/RemoteConfigClient.swift

```diff
@@ -135,7 +137,7 @@ public actor RemoteConfigClient: NSObject {
 
         super.init()
 
-        Task {
+        initializationTask = Task {
```

Comment:
is there a way we could assign to `fetchRemoteTask` so we don't have to introduce / check another task?